ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -223,22 +226,18 @@ compilerDependencies ghcFlavor =
     , "primop-vector-tys.hs-incl"
     , "primop-vector-uniques.hs-incl"
     ] ++
-    [ x | ghcFlavor == GhcMaster
-        , x <- [ "primop-docs.hs-incl"
-               , "GHC/Platform/Constants.hs"
-               ]
-    ]
+    [ "primop-docs.hs-incl" | ghcFlavor `elem` [ GhcMaster, Ghc901 ] ]
```

Comment:
What happened with `Constants.hs`?

Reply:
It was being listed twice (and I judged was out of place in) `compilerDependencies`. It remains in `packageCode`.